packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -2365,15 +2367,15 @@ describe('postgresql-multi-schema', () => {
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                              // - Enum "base_status"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                              // - Enum "transactional_status"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                              // - Model "base_some_table"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                              // - Model "transactional_some_table"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                              // 
-                                                                                                                                                                                                                                                                                                                    `)
+      // *** WARNING ***
+      // 
+      // These items were renamed due to their names being duplicates in the Prisma Schema Language:
```

Comment:
Unrelated: "in the Prisma Schema Language" is not really correct, right? It should say "in this Prisma schema"?

Reply:
+1